modules/500-upmeter/images/upmeter/pkg/probe/group_monitoring_and_autoscaling.go

```diff
@@ -39,7 +39,7 @@ func initMonitoringAndAutoscaling(access kubernetes.Access) []runnerConfig {
 				Access:                    access,
 				Timeout:                   5 * time.Second,
 				Namespace:                 "d8-monitoring",
-				LabelSelector:             "app=prometheus",
+				LabelSelector:             "app.kubernetes.io/name=prometheus",
```

Comment:
Unfortunately, this label selector matches `prometheus-longterm` pods, which should be not tracked by this probe. This probe should follow only pods of `prometheus-main`. How can we distinguish them?

In this branch: 

```shell
$ kubectl -n d8-monitoring get po -l app.kubernetes.io/name=prometheus
NAME                    READY   STATUS    RESTARTS   AGE
prometheus-longterm-0   3/3     Running   0          4m48s
prometheus-main-0       3/3     Running   0          3m58s
```

Reply:
```
$ kubectl -n d8-monitoring get po -l app.kubernetes.io/name=prometheus,operator.prometheus.io/name=main
NAME                READY   STATUS    RESTARTS   AGE
prometheus-main-0   3/3     Running   0          14d
```
?

Reply:
`operator.prometheus.io/name=main` one is enough. Tnahks, colleagues.